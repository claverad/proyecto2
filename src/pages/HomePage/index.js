import React from 'react'
import Header from '../Header/index';

const HomePage = () => (
  <div>
    <div><Header /></div>
  <div align="center" padding="20px">
    <h2>
      Bienvenid@, 
    </h2>
    <img src="https://rickandmortyapi.com/api/character/avatar/1.jpeg"/>
  </div>

  </div>
  
)

export default HomePage
