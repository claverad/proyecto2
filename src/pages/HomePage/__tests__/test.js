import React from 'react'
import HomePage from '../index'
import renderer from 'react-test-renderer'

it('Se renderizÃ³ el homePage', () => {
  const ss = renderer.create(
    <HomePage />
  ).toJSON()

  expect(ss).toMatchSnapshot()
})
