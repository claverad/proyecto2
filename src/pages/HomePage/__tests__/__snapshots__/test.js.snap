// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Se renderiz√≥ el homePage 1`] = `
<div>
  Soy el home 2
</div>
`;
